redux/profileSlice.ts

```diff
@@ -70,6 +72,18 @@ export const profileSlice: Slice = createSlice({
         };
       }
     },
+
+    setCreatedAt: (
```

Comment:
i think this should be good to leave in. we just want hasura to automatically set the created_at value, and then you can query it and set it here like you're already doing